sormas-ui/src/main/java/de/symeda/sormas/ui/email/ExternalEmailController.java

```diff
@@ -140,23 +150,44 @@ public <T extends HasUuid> void sendBulkEmail(
 				null,
 				Strings.messageBulkEmailsNoEligible,
 				Strings.messageBulkEmailsFinishedWithSkips,
-				Strings.messageBulkEmailsFinishedWithoutSuccess)
-				.doBulkOperation(
-					selectedEntries -> {
-						try {
-							return FacadeProvider.getExternalEmailFacade()
-								.sendBulkEmail(options, selectedEntries.stream().map(mapToReference).collect(Collectors.toList()));
-						} catch (IOException e) {
-							throw new RuntimeException(e);
-						}
-					},
-					selectedEntitiesCpy,
-					bulkOperationDoneCallback);
+				Strings.messageBulkEmailsFinishedWithoutSuccess).doBulkOperation(selectedEntries -> {
+					try {
+						return FacadeProvider.getExternalEmailFacade()
+							.sendBulkEmail(options, selectedEntries.stream().map(mapToReference).collect(Collectors.toList()));
+					} catch (IOException e) {
+						throw new RuntimeException(e);
+					}
+				}, selectedEntitiesCpy, bulkOperationDoneCallback);
 
 			optionsPopup.close();
 			Notification.show(null, I18nProperties.getString(Strings.notificationExternalEmailSent), Notification.Type.TRAY_NOTIFICATION);
 		});
 
 		optionsCommitDiscard.addDiscardListener(optionsPopup::close);
 	}
+
+	@NotNull
+	private static <T extends HasUuid> List<ReferenceDto> getReferenceDtos(RootEntityType rootEntityType, Collection<T> selectionReference) {
```

Comment:
this method seems not to be used, so it can be deleted

Reply:
removed